src/Rules/Resources/Descriptions.Designer.cs

```diff
@@ -736,7 +736,7 @@ internal class Descriptions {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to The Name property for the given element is empty, but the element isn&apos;t focusable. Please consider whether or not the element should have a name..
+        ///   Looks up a localized string similar to The UI Automation Name property for the given element is empty, but the element isn&apos;t focusable. Please consider whether or not the element should have a name..
```

Comment:
Inserting `UI Auromation` here seems a bit strange, since we don't use it when referring to other UIA properties